src/ir/literal-utils.h

```diff
@@ -31,6 +32,12 @@ inline Expression* makeFromInt32(int32_t x, Type type, Module& wasm) {
 }
 
 inline Expression* makeZero(Type type, Module& wasm) {
+  // TODO: Switch to using v128.const once V8 supports it
```

Comment:
is there a bug number we can add in a link here?

do other impls support it? if so can mention that in the comment too, as when we change this may depend on those too.

Reply:
Added a link. All other implementations that implement any SIMD implement this instruction (https://github.com/WebAssembly/simd/blob/master/proposals/simd/ImplementationStatus.md), so we should only have to worry about V8.